src/script/team/TeamRepository.ts

```diff
@@ -127,19 +126,25 @@ export class TeamRepository extends TypedEventEmitter<Events> {
     );
   };
 
-  initTeam = async (
-    teamId?: string,
-  ): Promise<{team: TeamEntity; members: QualifiedId[]} | {team: undefined; members: never[]}> => {
+  /**
+   * Will init the team configuration and all the team members from the contact list.
+   * @param teamId the Id of the team to init
+   * @param contacts all the contacts the self user has, team members will be deduced from it.
+   */
+  async initTeam(teamId: string, contacts: User[] = []): Promise<TeamEntity | undefined> {
     const team = await this.getTeam();
     // get the fresh feature config from backend
     await this.updateFeatureConfig();
     if (!teamId) {
-      return {team: undefined, members: []};
+      return undefined;
```

Comment:
why are we changing the return type?

Reply:
Oh that's because we do not need the members anymore from the consuming side :)